lib/models/messages/twitch/message.dart

```diff
@@ -182,6 +182,10 @@ class TwitchMessageModel extends MessageModel {
 
   bool get isCommand => !isAction && message.startsWith("!");
 
+  bool get isVip => badges.any((element) => element.key == 'vip');
+
+  bool get isMod => badges.any((element) => element.key == 'moderator');
```

Comment:
I think IRC already sends a property telling if the user has the [mod badge](https://dev.twitch.tv/docs/irc/tags#usernotice-twitch-tags).